packages/Webkul/Shop/src/Resources/views/components/layouts/header/mobile/index.blade.php

```diff
@@ -34,8 +34,60 @@
                 </x-slot:header>
 
                 <x-slot:content>
+                    {{-- Localization & Currency Section --}}
+                    <div class="flex w-full items-center mb-[15px] gap-x-[20px] justify-between">
+
+                        <x-shop::dropdown position="bottom-left">
+                            <!-- Dropdown Toggler -->
+                            <x-slot:toggle >
+                                <div class="flex items-center justify-between w-full gap-[10px] cursor-pointer">
+                                    <span>
+                                        {{ core()->getCurrentCurrency()->symbol . ' ' . core()->getCurrentCurrencyCode() }}
+                                    </span>
+
+                                    <span class="text-[24px] icon-arrow-down"></span>
```

Comment:
Hi @JunaidJaved248 ,
Please follow the standard sequence for CSS classes that we are using. [Take a Reference](https://tailwindcss.com/blog/automatic-class-sorting-with-prettier)

Reply:
Okay Right @amit-webkul